### DISCUSSION TITLE
help with entities

### DISCUSSION THREAD

**Discussion by Subia75** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#530] [Q&A] [answered]
hi
I ask if there is the possibility to detect the opening of the lid of the tank, or the absence of the tank in the vacuum?
I have an automation for emptying, which sends the vacuum to a defined place and waits, then once emptied I send it home by pressing the button, but I would like to automate it once the tank has been replaced.

Also I'd like to figure out how to find the states the attributes or even the entities that the vacuum exposes but I don't know how to.

I ask to understand how to do it, if I can have an example to recover a sensor.

thanks

**Comment by PiotrMachowski** [comment, marked as answer]
You have to ask this question in the repository of the integration that you use to connect your vacuum to HA.
